### DISCUSSION TITLE
숫자 고정폭 버전은 없는지요?

### DISCUSSION THREAD

**Discussion by StephenKwon** [orioncactus/pretendard#110] [Q&A] [answered]
모든걸 만족하며 사용하고 있지만, 단 한가지, 숫자가 폭이 모두 균일한 고정폭이었으면 하는 바램이 있습니다.

혹시 고정폭 버전은 없는지요?

**Comment by orioncactus** [comment, marked as answer]
Pretendard에서는 이미 고정폭 숫자 기능을 지원합니다. 사용하시는 환경에서 OpenType Feature를 지원한다면, 해당 환경에서 `tnum`을 적용해 활성화하시면 됩니다. 웹에서는 다음과 같이 코드를 추가해 고정폭 숫자를 사용하실 수 있습니다.
- `font-feature-settings: 'tnum';`

**Reply by StephenKwon** [reply]
빠른 회신에 감사드립니다. 윈도우 환경에서 기본 시스템 글꼴로 사용중입니다. 말씀주신 내용은 웹폰트에서의 옵션 적용인 것으로 보이는데요, 혹시 윈도우 PC 환경에서 고정폭 세팅을 하는 방법은 없는지요? 1.3.6 버전의 static 폴더 TrueTypeFont 를 설치하였습니다. 감사합니다.

**Reply by orioncactus** [reply]
Windows에서 시스템 글꼴을 Pretendard로 설정하신 것이라면, Windows에서는 tnum 기능을 기본으로 활성화하도록 설정할 수 없기 때문에, 고정폭 숫자 기능을 기본으로 적용하도록 수정한 Pretendard 글꼴 파일을 생성해야 합니다. 다행히도 특정 OpenType 기능을 글꼴 자체에 적용할 수 있는 도구를 제공하는 [fonttools-opentype-feature-freezer](https://github.com/twardoch/fonttools-opentype-feature-freezer)라는 프로젝트가 있는데요, 해당 프로젝트에서 애플리케이션을 다운로드해 사용하시는 것을 추천드립니다.

opentype-feature-freezer GUI 애플리케이션을 사용해 고정폭 기능이 기본으로 적용된 Pretendard를 생성하시려면, 해당 애플리케이션에서 options to control feature 섹션에 있는 features 부분에 `tnum`을 적고 Start 버튼을 누르시면 됩니다.

**Reply by StephenKwon** [reply]
알려주신 방법대로 MAC을 통해 변환하여 사용할 수 있게 되었습니다. PC버전은 링크가 죽었더라고요. 감사합니다!

**Reply by reteu5** [reply]
윈도우 버전용 링크도 남아있길래 남겨놓고 갑니다. (Windows 11 작동 확인)
https://github.com/twardoch/fonttools-opentype-feature-freezer/issues/22
